lxc_test.go

```diff
@@ -1051,6 +1051,36 @@ func TestIPAddress(t *testing.T) {
 	}
 }
 
+func TestIPv4Address(t *testing.T) {
+	c, err := NewContainer(ContainerName)
+	if err != nil {
+		t.Errorf(err.Error())
+	}
+
+	if unprivileged() {
+		time.Sleep(3 * time.Second)
```

Comment:
I merged it anyway but I really don't like sleep calls with random intervals like that. Is there a reason not to use WaitIPAddresses here?